src/server/routes/launchpad.js

```diff
@@ -20,16 +20,18 @@ router.post('/launchpad/snaps', newSnap);
 router.get('/launchpad/snaps', findSnap);
 
 router.use('/launchpad/snaps/list', json());
+router.use('/launchpad/snaps/list', verifyToken);
```

Comment:
I'd previously been adding `verifyToken` to POST requests such as `/launchpad/snaps/request-builds` which writes to resources. But [this comment](https://github.com/canonical-ols/build.snapcraft.io/pull/707#discussion_r114337414) has made me think we should protect endpoints that offer privileged information too.